tests/baselines/reference/user/acorn.log

```diff
@@ -62,6 +62,8 @@ node_modules/acorn/dist/acorn.es.js(3025,38): error TS2339: Property 'exprAllowe
 node_modules/acorn/dist/acorn.es.js(3026,14): error TS2339: Property 'value' does not exist on type 'TokenType'.
 node_modules/acorn/dist/acorn.es.js(3026,40): error TS2339: Property 'inGeneratorContext' does not exist on type 'TokenType'.
 node_modules/acorn/dist/acorn.es.js(3029,8): error TS2339: Property 'exprAllowed' does not exist on type 'TokenType'.
+node_modules/acorn/dist/acorn.es.js(3590,26): error TS2339: Property 'validFlags' does not exist on type 'RegExpValidationState'.
```

Comment:
some members are not resolved outside RegExpValidationState methods anymore. The language service also seems to be unstable inside methods -- if you get quick info on `this.flags/**/`, you get nothing, but get quick info on `this/**/`.flags` then `this.flags/**/` and you get the correct symbol.

Reply:
Filed #33013 to track this.